DEV_GUIDE.md

```diff
@@ -295,15 +295,14 @@ rm foo.txt .fossa.yml
 
 It's critically important to ensure that you have the correct JDK v8 installed before you can successfully compile the Stargate project. There are a number of versions of JDK 8 and a number of different ways to install them, but not all of them will work successfully with Stargate.
 
-Download and install Maven from this link: https://maven.apache.org/download.cgi
+Download and install Maven from this link: https://maven.apache.org/download.cgi or use the included [maven wrapper](https://github.com/stargate/stargate/blob/master/mvnw) script as you would ordinarily use the `mvn` command.
 
 Download the JDK from this link: https://adoptopenjdk.net/?variant=openjdk8&jvmVariant=hotspot
 
-Install that version of te JDK and then be sure to add it to your path. If you are using a newer version of the Mac OS, then you are likely using Z-Shell by default. So open your ~/.zshrc file and add the path there. Here is an example:
+Install that version of the JDK and then be sure to add it to your path. If you are using a newer version of the Mac OS, then you are likely using Z-Shell by default. So open your ~/.zshrc file and add the path there. Here is an example:
 
 ```sh
 export JAVA_HOME="/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home"
-export MAVEN_HOME="/Users/<your user name>/development/apache-maven-3.8.1"
 export PATH="$MAVEN_HOME/bin:$JAVA_HOME/bin:$PATH"
```

Comment:
Should the PATH export be removed if `MAVEN_HOME` not specified either?

Reply:
Good catch